package.json

```diff
@@ -10,10 +13,13 @@
     "node": "12"
   },
   "scripts": {
-    "start": "node lib",
-    "dev": "node lib | ./node_modules/.bin/bunyan",
-    "test": "npm run lint && CI=1 tap -R spec test/**/*.test.js --timeout 60",
-    "lint": "eslint -c .eslintrc cli lib test && ./lintVerifier.sh",
+    "build": "tsc",
+    "start": "node dist",
+    "dev": "tsc-watch --project tsconfig.json --onSuccess 'node .' | ./node_modules/.bin/bunyan",
+    "test": "npm run lint && CI=1 tap -R spec test/**/*.test.[tj]s --timeout 60",
+    "lint": "npm run lint:js && npm run lint:ts",
+    "lint:ts": "tslint --project tsconfig.json --format stylish",
```

Comment:
`eslint` now does TS, as seen in the `.eslintrc.js` you added, shouldn't need `tslint`.

`tslint` is deprecated: https://github.com/palantir/tslint/issues/4534

Reply:
Thanks for confirming, this one confused me abit. Think the Notion doc might be a little out of date so will update that too.